packages/core/src/html/SiteLinkManager.ts

```diff
@@ -69,4 +74,48 @@ export class SiteLinkManager {
     this._addToCollection(resourcePath, cwf);
     return 'Intralink collected to be validated later';
   }
+
+  maintainFilePathToHashesMap(node: MbNode, cwf: string, forceWrite: string = '') {
```

Comment:
Could you add a comment explaining what is forceWrite and when should it be used?

Reply:
Method document added in later commit.